src/cljs/main/broadfcui/page/workspace/monitor/workflow_details.cljs

```diff
@@ -235,26 +235,24 @@
           input-names (string/split inputs ".")
           workflow-name (first input-names)]
       (create-field "Workflow ID"
-                    (links/create-external {:href (str moncommon/google-cloud-context
+                    (links/create-external {:href (str moncommon/google-storage-context
                                                        bucketName "/" submission-id "/"
                                                        workflow-name "/" (workflow "id") "/")}
                       (workflow "id"))))
     (let [status (workflow "status")]
       (create-field "Status" (moncommon/icon-for-wf-status status)))
     (let [call-cache-status (-> (workflow "calls") vals first first (get "callCaching") (get "effectiveCallCachingMode"))]
       (create-field "Call Caching" (moncommon/call-cache-result call-cache-status)))
-    (when (workflow "submission")
-      (create-field "Submitted" (moncommon/render-date (workflow "submission"))))
-    (when (workflow "start")
-      (create-field "Started" (moncommon/render-date (workflow "start"))))
-    (when (workflow "end")
-      (create-field "Ended" (moncommon/render-date (workflow "end"))))
+    (when-let [submission (workflow "submission")]
+      (create-field "Submitted" (moncommon/render-date submission)))
+    (when-let [start (workflow "start")]
+      (create-field "Started" (moncommon/render-date start)))
+    (when-let [end (workflow "end")]
+      (create-field "Ended" (moncommon/render-date end)))
     [IODetail {:label "Inputs" :data (utils/parse-json-string (get-in workflow ["submittedFiles", "inputs"]))}]
     [IODetail {:label "Outputs" :data (workflow "outputs")}]
-    [:div {:style {:whiteSpace "nowrap" :marginRight "0.5em"}}
-     (let [wlogurl (str "gs://" bucketName "/" submission-id "/workflow.logs/workflow."
-                        (workflow "id") ".log")]
-       (create-field "Workflow Log" (display-value wlogurl (str "workflow." (workflow "id") ".log"))))]
+    (when-let [workflowLog (workflow "workflowLog")]
```

Comment:
we don't need to construct this because Cromwell tells us directly

Reply:
this `workflowLog` also removes a `div` - intentional?